.github/scripts/ci_tests.sh

```diff
@@ -11,20 +11,27 @@ TEST_RUN_E2E_TESTS=${1:-1}
 TEST_CORES=${2:-2}
 TEST_JOB_COUNT=${3:-2}
 
-echo -e "Building tests"
-cmake tests -Bbuild
+echo -e "::group::Building unit tests"
```

Comment:
I see that currently we are mixing the 32 and 64 bit arch
https://github.com/Azure/azure-iot-middleware-freertos/pull/96/checks?check_run_id=2449924893#step:5:652

Can we make our cmake agnostic to this and let ci_test.sh to only select 32 bit now? 
-m32 flag for gcc help you compile 32bit binaries.

Reply:
is that not a `node` problem?

Reply:
I should not be ... node exec the binary and binary that should be build should be of 32 bin .... looks like currently binary is getting build with x64. Not sure why, as in early run logs (current master) I am not seeing this type of loading error.

Reply:
I don't think this is introduced with this PR. [Link to a previous one with the same one.](https://github.com/Azure/azure-iot-middleware-freertos/runs/2416518140?check_suite_focus=true#step:5:433) We can look into it but I don't think it's to be fixed here.

Reply:
ooh then its bit more then how you are building it here . I can take a look into this.